unity/ARUI/ProjectSettings/ProjectSettings.asset

```diff
@@ -705,7 +705,8 @@ PlayerSettings:
   gcIncremental: 1
   assemblyVersionValidation: 1
   gcWBarrierValidation: 0
-  apiCompatibilityLevelPerPlatform: {}
+  apiCompatibilityLevelPerPlatform:
+    WSA: 3
```

Comment:
What is this settings change?

Reply:
That is the change to use .Net API level 4.x (supposed to include more up to date APIs). Our original Unity/Visual Studio installation with the versions we recorded came with .Net 5.x (which is now outdated anyway in lieu of 6.x/7.x). I made this change initially to try to support the APIs now commented out, but didn't actually work, but seemed like a sensible thing to do given the mentioned context.